easybuild/easyconfigs/l/libunistring/libunistring-0.9.7-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
```

Comment:
I'm not clear on what I need to change here and why.  Any additional info appreciated.